go.mod

```diff
@@ -152,11 +152,11 @@ require (
 	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce // indirect
 	gopkg.in/ory-am/dockertest.v3 v3.3.4 // indirect
 	gopkg.in/square/go-jose.v2 v2.3.1
-	gopkg.in/yaml.v2 v2.2.4
+	gopkg.in/yaml.v2 v2.2.7
 	istio.io/api v0.0.0-20200107183329-ed4b507c54e1
 	istio.io/gogo-genproto v0.0.0-20191029161641-f7d19ec0141d
 	istio.io/operator v0.0.0-20200128164134-d8ab51c0a823
-	istio.io/pkg v0.0.0-20191030005435-10d06b6b315e
+	istio.io/pkg v0.0.0-20200206133456-cc9c9062dd4b
```

Comment:
can you revert this?

do you need to update istio.deps?

Reply:
Yep. I'm not sure why Intellij decided all these things needed to change and the only one I wanted changed (operator) got reverted. Fixing.